packages/calcite-components/src/components/table-row/table-row.scss

```diff
@@ -5,11 +5,9 @@
  *
  * @prop --calcite-table-row-background: [Deprecated] Use `--calcite-table-row-background-color` instead. Specifies the background color of the component.
  * @prop --calcite-table-row-background-color: Specifies the background color of the component.
- * @prop --calcite-table-row-background-color-striped: The background color of the component's `striped` rows, when specified.
+ * @prop --calcite-table-row-secondary-background-color: The background color of the component's `striped` rows, when specified.
  * @prop --calcite-table-row-background-color-selected: The background color of the component's `selected` rows, when specified.
  * @prop --calcite-table-row-border-color: Specifies the border color of the component.
- * @prop --calcite-table-row-selected-accent-color: Specifies the border color of the component.
```

Comment:
Unused

Reply:
Can we keep the accent color terminology vs. moving to shadow? The fact the accent border is implemented via a box shadow shouldn’t matter to a user IMO. It’s really changing the accent color.